Fix build error on macOS:

error: use of undeclared identifier '__sighandler_t'
--- rzip64.h.orig	2010-12-19 11:48:44.000000000 -0600
+++ rzip64.h	2020-11-13 08:17:12.000000000 -0600
@@ -120,6 +120,9 @@
 #define TRUE 1
 #endif
 
+#ifndef __sighandler_t
+#define __sighandler_t sig_t
+#endif
 
 #if !HAVE_STRERROR
 extern char *sys_errlist[];
